### DISCUSSION TITLE
cannot install rustdesk on windows 10

### DISCUSSION THREAD

**Discussion by ardeal** [rustdesk/rustdesk#3965] [Q&A]
Hi, 
I am installing rustdesk on my windows 10, but I am experiencing the following issue:
my windows 10 version:
22H2, 
OS build: 190452728

when I double click rustdesk 32bit or rustdesk 64bit, I encountered the same issue:
Windows cannot access the specified device, path, or file. you may not have the appropriate permissions to access the item.
![Capture](https://user-images.githubusercontent.com/6227348/230703185-4345b15d-65e0-48b8-a803-691fb6410083.PNG)

the problem is: I login windows 10 with admin user, and I could install any other software on it.

**Comment by cyber-sec0** [comment]
It can be bypassed and the app works if you install the app on a Windows 11 machine (like a virtual machine), copy the "C:\Program Files\RustDesk" folder, then paste it on the Windows 10 machine you want to use RustDesk on

**Reply by kozman1000** [reply]
Additionally, have you tried updating from March 2023 security patch level 19045.2728 to the December 2023 19045.3803 Security patch level? Could be something there causing your issue.
